examples/plotting/glyphs.py

```diff
@@ -1,3 +1,18 @@
+'''This example shows a various types of Glyphs that are offered by bokeh library.
+Such as annular wedge, annulus, arc, bezier, circle, ellipse, Hbar, line etc.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.annular_wedge, bokeh.plotting.figure.annulus,
+    bokeh.plotting.figure.arc, bokeh.plotting.figure.bezier, bokeh.plotting.figure.circle,
+    bokeh.plotting.figure.ellipse, bokeh.plotting.figure.hbar, bokeh.plotting.figure.line,
+    bokeh.plotting.figure.multi_line, bokeh.plotting.figure.multi_polygon,
+    bokeh.plotting.figure.patch, bokeh.plotting.figure.patches, bokeh.plotting.figure.quad,
+    bokeh.plotting.figure.quadratic, bokeh.plotting.figure.ray, bokeh.plotting.figure.rect,
+    bokeh.plotting.figure.scatter
```

Comment:
The extension directive is not smart enough to handle multi-lines. We can only have a single line of `:apis:` They can be fairly long line, within reason, even using `# noqa` This is probably too many. Please pare down to the most interesting/important and put on a single line (here and below)

Reply:
Did the required changes.